lightningd/Makefile

```diff
@@ -39,24 +39,26 @@ LIGHTNINGD_SRC :=				\
 	lightningd/subd.c			\
 	lightningd/watch.c
 
-
 LIGHTNINGD_SRC_NOHDR :=				\
 	lightningd/datastore.c			\
 	lightningd/offer.c			\
 	lightningd/signmessage.c
 
-LIGHTNINGD_HEADERS :=				\
+include wallet/Makefile
+
+LIGHTNINGD_HDRS :=				\
```

Comment:
Why change the `LIGHTNINGD_HEADERS` to this more cryptic `LIGHTNINGD_HDRS` stuff?

Reply:
This name change was requested by @rustyrussell and updated accordingly.

Reply:
Ops! My fault I missed this review sorry! Ignore it